op-service/txmgr/txmgr.go

```diff
@@ -604,9 +604,14 @@ func (m *SimpleTxManager) checkLimits(tip, basefee, bumpedTip, bumpedFee *big.In
 }
 
 // calcThresholdValue returns x * priceBumpPercent / 100
+// It guarantees that x is increased by at least 1
 func calcThresholdValue(x *big.Int) *big.Int {
 	threshold := new(big.Int).Mul(priceBumpPercent, x)
-	threshold = threshold.Div(threshold, oneHundred)
+	threshold.Div(threshold, oneHundred)
+	// Guarantee to add at least 1 wei. Edge-case during near-zero fee conditions.
+	if threshold.Cmp(x) == 0 {
+		threshold.Add(threshold, common.Big1)
+	}
```

Comment:
nit: 
```suggestion
	if threshold.Cmp(x) == 0 {
		threshold = big.New(1)
	}
```

I'm somewhat scared of using `common.Big1` after having discovered it's actually mutable and sometimes winds up not actually being 1, but either way we can probably just set the value rather than doing an addition for simplicity.

Reply:
Will change to its own instance of 1 👍🏻